app/uk/gov/hmrc/economiccrimelevyregistration/views/UtrTypeView.scala.html

```diff
@@ -34,16 +34,12 @@
             @errorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
         }
 
-        <h1 class="govuk-heading-l">@messages("utrType.heading")</h1>
-
-        <p class="govuk-body">@messages("utrType.p1")</p>
-
```

Comment:
Why did you remove those?

Reply:
Because it shouldn't be there. When you set a legend to asPageHeading, it creates the H1 for you.

In addition, hinttext takes the place of that paragraph.